nixos/modules/services/misc/mesos-slave.nix

```diff
@@ -43,6 +51,12 @@ in {
         type = types.bool;
       };
 
+      withDocker = mkOption {
```

Comment:
@wmertens That assumes you want to run the docker containerizer as soon as you also have the docker service enabled, which is not necessarily true (not in my case anyway). Is there a more elegant nix way to do this so that:

- default is to enable docker containerizer if docker service is enabled
- that behaviour is overridable, ie. you can force disabling docker containerizer